java-compute/google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/Util.java

```diff
@@ -0,0 +1,37 @@
+package com.google.cloud.compute.v1.integration;
+
+import com.google.cloud.compute.v1.DeleteInstanceRequest;
+import com.google.cloud.compute.v1.Instance;
+import com.google.cloud.compute.v1.InstancesClient;
+import com.google.cloud.compute.v1.InstancesClient.ListPagedResponse;
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+
+public class Util {
+
+  // Cleans existing test resources if any.
+  private static final int DELETION_THRESHOLD_TIME_HOURS = 24;
+
+  /** Bring down any instances that are older than 24 hours */
+  public static void cleanUpComputeInstances(
+      InstancesClient instancesClient, String project, String zone) {
+    ListPagedResponse listPagedResponse = instancesClient.list(project, zone);
+    for (Instance instance : listPagedResponse.iterateAll()) {
+      if (isCreatedBeforeThresholdTime(
+              ZonedDateTime.parse(instance.getCreationTimestamp()).toInstant())
+          && instance.getName().startsWith(BaseTest.COMPUTE_PREFIX)) {
```

Comment:
For both `Util` classes, I think I would rather have the prefix passed in as an argument into this method, but I guess this is OK too.

Reply:
On second thought, this seems better. I'll create a new PR for this.